pkg/controller/v1/xgboostjob/pod.go

```diff
@@ -58,7 +59,7 @@ func convertPodList(list []corev1.Pod) []*corev1.Pod {
 	return ret
 }
 
-// Set the pod env set for XGBoost Rabit Tracker and worker
+// SetPodEnv sets the pod env set for XGBoost Rabit Tracker and worker
```

Comment:
Update comment to include LightGBM as well

Reply:
[Added](https://github.com/kubeflow/xgboost-operator/pull/103/commits/d90f9452746cec231de25352029094043b127c4b)